app-text/ghostwriter/ghostwriter-1.7.3.recipe

```diff
@@ -5,7 +5,7 @@ COPYRIGHT="wereturtle et al."
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/wereturtle/ghostwriter/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="bfd64024d9741f55d587afa5661f4b5e2a48becbbc92495c2186fca44fb08486"
+CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
 ADDITIONAL_FILES="ghostwriter.rdef.in"
```

Comment:
```diff
 CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
+SOURCE_FILENAME="ghostwriter-$portVersion.tar.gz"
 ADDITIONAL_FILES="ghostwriter.rdef.in"
```

This change is to make the recipe conform to our [guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name

Reply:
Done